guides/source/active_model_basics.md

```diff
@@ -285,101 +800,308 @@ irb> person = Person.new
 irb> person.token = "2b1f325"
 irb> person.valid?
 => false
-irb> person.name = 'vishnu'
-irb> person.email = 'me'
+
+irb> person.name = "vishnu"
+irb> person.email = "me"
 irb> person.valid?
 => false
-irb> person.email = 'me@vishnuatrai.com'
+
+irb> person.email = "me@vishnuatrai.com"
```

Comment:
I feel that for this contributor's sake, we shouldn't use his real email or URL, and all emails and URLs we do use should be dummies. (He might not mind himself, but let's not give the bots anything worthwhile to scrape.)